gz_ros2_control/CMakeLists.txt

```diff
@@ -24,39 +24,41 @@ find_package(pluginlib REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(yaml_cpp_vendor REQUIRED)
 
-if("$ENV{IGNITION_VERSION}" STREQUAL "citadel")
```

Comment:
Just checking if you are removing Citadel support on purpose

Reply:
This packages uses `ros_gz` to spawn models, we can't install one the required version of `ros_gz_sim` which is `ros_ign_gazebo`.

Reply:
I'm not sure I understand. What ROS versions is this branch targeting?

Reply:
there are `foxy` and `galactic` branches